src/field.ml

```diff
@@ -96,7 +104,7 @@ module MakeCommon (Params : Params.S) : COMMON = struct
   end)
 
   module Pointer = MakeInt (struct
-    let minimal_size = Params.page_sz |> Fmt.str "%x" |> String.length |> fun x -> (x + 1) / 2
+    let minimal_size = (Params.page_sz - 1) |> Fmt.str "%x" |> String.length |> fun x -> (x + 1) / 2
```

Comment:
Just a guess

Reply:
You're probably right since the pointer to the last addressable byte should be `Params.page_sz - 1`